### ISSUE TITLE
Bug: Mismatching type of difficulty_trend in deprecated active_difficulty call

### ISSUE THREAD

**Issue by keerifox** [nanocurrency/nano-node#3323]
### Summary

`difficulty_trend` returns a `string` rather than an `array`, which caused issues in some wallets relying on it, e.g gonano wallet:

> json: cannot unmarshal string into Go struct field .diffulty_trend of type []string

### Node version

Nano V22.0

### Build details

several community-ran nodes

### OS and version

several community-ran nodes

### Steps to reproduce the behavior

make an RPC call
```json
{
  "action": "active_difficulty",
  "include_trend": "true"
}
```

### Expected behavior

expected to produce a list as per [docs page](https://docs.nano.org/commands/rpc-protocol/#active_difficulty)

![image](https://user-images.githubusercontent.com/29272208/120865382-75086380-c57d-11eb-8bee-5a4e9526f207.png)

### Actual behavior

produces an empty string instead
```json
"difficulty_trend": ""
```

### Possible solution

return a single-item list
```json
"difficulty_trend": [
  "1.000000000000000"
]
```

**Comment by shryder** [comment]
This RPC command is now marked as deprecated.

**Comment by keerifox** [comment]
that's what the title of this issue says yes

methods are deprecated to allow to transition away from using them, if there's a breaking change that causes existing integrations to malfunction then it defeats the purpose

**Comment by shryder** [comment]
My bad, didn't notice it in the title.

**Comment by shryder** [comment]
Looks like the RPC was [completely removed](https://github.com/nanocurrency/nano-node/commit/9982779c1bbf8c108ade266352f62abab5ca9083#diff-29873a42cf5fc94715ac705d456a6f7a762ff89b1d5109bae53f86002e5a6935L1012) and then got [re-added in](https://github.com/nanocurrency/nano-node/commit/69f7f523ac24829bee8db71c04405d9471faee29#diff-29873a42cf5fc94715ac705d456a6f7a762ff89b1d5109bae53f86002e5a6935R1027) but now will always send an empty difficulty_trend response (RPC always returns empty string instead of empty array). I can make a pull request to make it send `["1.000000000000000"]` back

**Comment by zhyatt** [comment]
Fixed by #3343
